### PR TITLE
Translate readme to English

### PR DESCRIPTION
**Pull request by paphonb**
## Why did you create this PR
- The readme page proposed by @saenyakorn is in Thai, while almost everything else is in English

## What did you do
- Translated the readme to English and adjusted it as I see fit

## Demo
[https://github.com/thinc-org/cugetreg-frontend/blob/pun/docs/readme/README.md](https://github.com/thinc-org/cugetreg-frontend/blob/pun/docs/readme/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,84 +2,80 @@
...
-
-1. [สำนักงานการทะเบียน จุฬาลงกรณ์มหาวิทยาลัย](https://www.reg.chula.ac.th) - ผู้สนับสนุน Server ให้กับเว็บไซต์ CU Get Reg
-
-2. [ศูนย์การศึกษาทั่วไป จุฬาลงกรณ์มหาวิทยาลัย](https://www.gened.chula.ac.th) - ผู้สนับสนุนข้อมูลคำอธิบายรายวิชา GenEd และเป็นผู้สนับสนุนข้อมูลตอนเรียนวิชา GenEd
+## Sponsors
 
-3. [Gen อย่าได้ Ed](https://www.facebook.com/genedahs) - ร่วมมือกับ CU Get Reg ในการประชาสัมพันธ์เว็บไซต์ให้กับนิสิตจุฬาฯ
+1. [Office of the Registrar - Chulalongkorn University](https://www.reg.chula.ac.th) - provides the infrastructure for CU Get Reg
+2. [General Education Center - Chulalongkorn University](https://www.gened.chula.ac.th) - provides course description for GenEd courses
+3. [Gen อย่าได้ Ed](https://www.facebook.com/genedahs) - helps promote our website
```
Comment (natTP) [inline README.md:81]:
```suggestion
3. [Gen อย่าได้ Ed](https://www.facebook.com/genedahs) - helps promote our website and provides data on GenEd courses
```

File: README.md [hunk 2]
```diff
@@ -2,84 +2,80 @@
...
+- [Contacts](#contacts)
+- [Sponsors](#sponsors)
 
 <div id="beginning-of-cu-get-reg"></div>
 
-## จุดเริ่มต้นของ CU Get Reg
+## The beginning of CU Get Reg
 
-    "GenEd เทอมนี้เราจะลงวิชาอะไรดีนะ?"
+    "Which GenEd course do I enroll this semester?"
```
Comment (natTP) [inline README.md:21]:
```suggestion
    "Which GenEd courses should I enroll this semester?"
```

File: README.md [hunk 3]
```diff
@@ -2,84 +2,80 @@
...
 
-    "GenEd ที่เราเลือกลงไปเวลาจะไปทับกับวิชาบังคับหรือเปล่านะ?"
+    "Will this GenEd course conflict with my compulsary courses?"
```
Comment (natTP) [inline README.md:23]:
```suggestion
    "Will this course conflict with my other courses?"
```
I don't think this applies to just GenEds, and I want people to be aware that CUgetreg focuses on more than just GenEds

File: README.md [hunk 4]
```diff
@@ -2,84 +2,80 @@
...
 
-    "จัดตารางเรียนยังไรดีนะ? วันเรียนในแต่ละ Section ก็ไม่เหมือนกันอีก สับสนมาก ๆ"
+    "This course is taught on different day in each section. How do I choose?"
```
Comment (natTP) [inline README.md:25]:
```suggestion
    "Each section of this course is taught on different days. How do I choose?"
```

File: README.md [hunk 5]
```diff
@@ -2,84 +2,80 @@
...
 
-และอีกสารพัดคำถามมากมายท่ามกลางความสับสนว่าจะวางแผนการเรียนอย่างไรดีในภาคการศึกษานี้ แน่นอนว่า [ทีมผู้พัฒนา](#developers) ก็เป็นหนึ่งในนั้น
+And many other questions from many students trying to make their study plan for the semester. And we are one of them.
```
Comment (natTP) [inline README.md:27]:
```suggestion
And many other problems students face when planning their timetable for the semester. We, the CU Get Reg team, are no different.
```
Try not to repeat 'and' too much for more impact
Reordered words and changed words to be more concise.
`We are one of them` would be ambiguous between `we are questions` and `we are students`

File: README.md [hunk 6]
```diff
@@ -2,84 +2,80 @@
...
 
-การวางแผนลงทะเบียนในแต่ละเทอมเมื่อก่อนยุ่งยากและเสียเวลามาก ๆ ข้อมูลวิชาเรียนกระจัดกระจาย โดยเฉพาะอย่างยิ่งวิชา GenEd การจะหาวิชา GenEd ที่ใช่ เวลาเรียนที่ลงตัว ต้องหาข้อมูลจากหลากหลายแหล่งเพื่อประกอบการตัดสินใจ อีกทั้งยังมีเรื่องที่นั่งและความนิยมของวิชา GenEd เป็นปัจจัยสำคัญต่อการตัดสินใจ แต่ข้อมูลเหล่านั้นกลับเข้าถึงได้ยาก ทำให้วางแผนได้ยากลำบาก
+Picking courses and sections in each semester is a very complicated and time-consuming process. The course information are scattered, espescially GenEd courses. When trying to find the right GenEd, you have to consider many factors such as course contents, available seats, and popularity. But those information are hard to access, which makes planning very hard.
```
Comment (natTP) [inline README.md:29]:
```suggestion
Picking courses and sections in each semester is a very complicated and time-consuming process. Course information is scattered, espescially GenEd courses. When trying to find the right GenEd, you have to consider many factors such as course contents, available seats, and popularity. However, that information is hard to access which makes planning very difficult.
```

File: README.md [hunk 7]
```diff
@@ -2,84 +2,80 @@
...
 
-[ทีมผู้พัฒนา](#developers) เห็นถึงปัญหานี้ จึงได้วางแผนจัดทำเว็บไซต์เพื่ออำนวยความสะดวกในการวางแผนลงทะเบียนขึ้นมา
+We saw this problem and built [CU Get Reg](https://cugetreg.com) to make this planning process a breeze.
```
Comment (natTP) [inline README.md:31]:
```suggestion
We saw this problem and built [CU Get Reg](https://cugetreg.com) to make this planning process a breeze!
```
Not necessary, but would be a nice place to add an exclaimation point

File: README.md [hunk 8]
```diff
@@ -2,84 +2,80 @@
...
 <div id="related-repositories"></div>
 
-## โปรเจคที่เกี่ยวข้อง
+## Related repositories
 
-- [cugetreg-frontend](https://github.com/thinc-org/cugetreg-frontend) เว็บไซต์ CU Get Reg
-- [cugetreg-backend](https://github.com/thinc-org/cugetreg-backend) ระบบหลังบ้านของเว็บไซต์ CU Get Reg
-- [cugetreg-scraper](https://github.com/thinc-org/cugetreg-scraper) ระบบคัดลอกข้อมูลวิชาเรียนแบบสาธารณะจาก [Reg Chula](https://cas.reg.chula.ac.th/cu/cs/QueryCourseScheduleNew/index.html)
-- [chula-courses](https://github.com/thinc-org/chula-courses) Schema ส่วนกลางของข้อมูลรายวิชา
+- [cugetreg-frontend](https://github.com/thinc-org/cugetreg-frontend) The web interface that users see
```
Comment (natTP) [inline README.md:37]:
```suggestion
- [cugetreg-frontend](https://github.com/thinc-org/cugetreg-frontend) The web interface
```
Redundant? Especially considering this README's audience.

File: README.md [hunk 9]
```diff
@@ -2,84 +2,80 @@
...
+- [cugetreg-scraper](https://github.com/thinc-org/cugetreg-scraper) The system that fetches public course data from [Reg Chula](https://cas.reg.chula.ac.th/cu/cs/QueryCourseScheduleNew/index.html)
+- [chula-courses](https://github.com/thinc-org/chula-courses) Data schema
 
 <div id="contribution"></div>
 
-## มาเป็นส่วนหนึ่งกับเรา
+## Contribution
 
-พวกเรา [ทีมผู้พัฒนา](#developers) ยินดีต้อนรับนักพัฒนาทุกคน! มาร่วมเป็นส่วนหนึ่งในการพัฒนาสังคมของจุฬาลงกรณ์วิทยาลัยให้ดียิ่ง ๆ ขึ้นไปอีก! หากคุณส่วนรู้สึกสนใจที่จะมีส่วนร่วมกับเราแล้ว คุณสามารถเริ่มได้ที่นี่ได้เลย
+We welcome all developers. If you want to be a part of CU Get Reg and improve Chulalongkorn University society, you can start here!
```
Comment (natTP) [inline README.md:46]:
```suggestion
CU Get Reg is open source - we welcome all developers. If you want to be a part of CU Get Reg and improve Chulalongkorn University society, you can start here!
```
Would be nice to just directly state it

**Review by natTP — CHANGES_REQUESTED** [review]

File: README.md [hunk 10]
```diff
@@ -2,84 +2,80 @@
...
 
 - [Contribution Guides](#https://github.com/thinc-org/cugetreg-frontend/wiki/Contribution-Guides)
 - [Set up your local environments](https://github.com/thinc-org/cugetreg-frontend/wiki/Contribution-Guides)
 - [Explore issues](https://github.com/thinc-org/cugetreg-frontend/issues)
 - [Our Roadmap](https://github.com/thinc-org/cugetreg-frontend/wiki/Contribution-Guides)
 
-หากคุณยังรู้สึกลังเล คุณสามารถ [ติดต่อเรา](#contacts) เพื่อปรึกษาก่อนได้นะ
```
Comment (bombnp) [inline README.md:53]: not sure about this, feel free to ignore it if you think it's weird
```md
If you have any questions, feel free to [contact us](#contacts) to discuss potential contributions.
```

File: README.md [hunk 11]
```diff
@@ -2,84 +2,80 @@
...
 
 TODO
 
 <div id="contacts"></div>
 
-## ติดต่อเรา
+## Contacts
 
-คุณสามารถติดต่อเราได้ทุกเรื่องตลอดเวลาที่
+You can contact us anytime at
```
Comment (bombnp) [inline README.md:69]:
```suggestion
You can contact us anytime via
```

**Review by bombnp — CHANGES_REQUESTED** [review]
most things are already covered by @natTP . My suggestions are only built on top of hers

File: README.md (hunk 10 above)
Comment (paphonb) [reply README.md:53]: Idk about this, it feels weird

File: README.md [hunk 12]
```diff
@@ -2,84 +2,80 @@
...
 
-    "จัดตารางเรียนยังไรดีนะ? วันเรียนในแต่ละ Section ก็ไม่เหมือนกันอีก สับสนมาก ๆ"
+    "Each section of this course is taught on different days. How do I choose?"
 
-และอีกสารพัดคำถามมากมายท่ามกลางความสับสนว่าจะวางแผนการเรียนอย่างไรดีในภาคการศึกษานี้ แน่นอนว่า [ทีมผู้พัฒนา](#developers) ก็เป็นหนึ่งในนั้น
+And many other problems students face when planning their timetable for the semester. We, the CU Get Reg team, are no different.
 
-การวางแผนลงทะเบียนในแต่ละเทอมเมื่อก่อนยุ่งยากและเสียเวลามาก ๆ ข้อมูลวิชาเรียนกระจัดกระจาย โดยเฉพาะอย่างยิ่งวิชา GenEd การจะหาวิชา GenEd ที่ใช่ เวลาเรียนที่ลงตัว ต้องหาข้อมูลจากหลากหลายแหล่งเพื่อประกอบการตัดสินใจ อีกทั้งยังมีเรื่องที่นั่งและความนิยมของวิชา GenEd เป็นปัจจัยสำคัญต่อการตัดสินใจ แต่ข้อมูลเหล่านั้นกลับเข้าถึงได้ยาก ทำให้วางแผนได้ยากลำบาก
+Picking courses and sections in each semester is a very complicated and time-consuming process. Course information is scattered, espescially GenEd courses. When trying to find the right GenEd, you have to consider many factors such as course contents, available seats, and popularity. However, that information is hard to access which makes planning very difficult.
```
Comment (natTP) [inline README.md:29]:
```suggestion
Picking courses and sections in each semester is a very complicated and time-consuming process. Course information is scattered, especially GenEd courses. When trying to find the right GenEd, you have to consider many factors such as course contents, available seats, and popularity. However, that information is hard to access which makes planning very difficult.
```
1 small typo

**Review by natTP — CHANGES_REQUESTED** [review]

**Review by natTP — APPROVED** [review]
LGTM

**Review by bombnp — APPROVED** [review]

**Pull request merged**